lib/ansible/modules/monitoring/zabbix/zabbix_action.py

```diff
@@ -127,6 +127,7 @@
                 description:
                     - Condition operator.
                     - When I(type) is set to C(time_period), the choices are C(in), C(not in).
+                    - 'matches', 'does not match', 'Yes' and 'No' condition operators work only with >= Zabbix 4.0
```

Comment:
Failing because of:
```suggestion
                    - C(matches), C(does not match), C(Yes) and C(No) condition operators work only with >= Zabbix 4.0
```